src/components/SearchBar.js

```diff
@@ -0,0 +1,39 @@
+import {useState} from "react";
+import data from "../utils/data.json"
+
+
+/** search functionality to search memeber name and designation */
+const SearchMembers = (searchText) =>{
```

Comment:
You can make this a utility function and can add it to the utils folder. Also, name it in lowercase.